### PR TITLE
Support CentOS major release

### PR DESCRIPTION
**Pull request by lisibann**
These patches modify: directories from '7.2' to '7' , the directories at yaml files, gcc_release at gcc.spec to avoid error when installing gcc from hostos ex. required to build SLOF, kernel (bump to 12 which is bigger than 11 at distro 7.3).

### CODE REVIEW

**Comment by fabianorosas** [comment]
add to whitelist

**Comment by fabianorosas** [comment]
Folks, please review only the last two commits of this. The other ones are from #81 . We'll rebase afterwards.

File: gcc/centOS/7/gcc.spec [hunk 1]
```diff
@@ -3365,6 +3365,8 @@ fi
 %{_prefix}/libexec/gcc/%{gcc_target_platform}/%{gcc_version}/plugin
 
 %changelog
+* Tue Jan 03 2017 Lisiane M. B. Ambiel <lisianem@linux.vnet.ibm.com> - 4.8.5-12.2
```
Comment (fabianorosas) [inline gcc/centOS/7/gcc.spec]: Please remove the `.2` from the Release field and from this changelog entry. The whole version string should be only `4.8.5-12`. The `.2` was probably a mistake due to the use of `rpmdev-bumpspec` which doesn't understand variables in the Release field.

File: gcc/centOS/7/gcc.spec [hunk 2]
```diff
@@ -2,7 +2,7 @@
 %global SVNREV 225304
 # Note, gcc_release must be integer, if you want to add suffixes to
 # %{release}, append them after %{gcc_release} on Release: line.
-%global gcc_release 4
+%global gcc_release 12
```
Comment (olavphibm) [inline gcc/centOS/7/gcc.spec]: Minor: trailing whitespace

**Pull request merged**